app/views/staff/home/_delivery_partner_organisations_table.html.haml

```diff
@@ -11,9 +11,8 @@
     - @delivery_partner_organisations.each do |delivery_partner_organisation|
       %tr.govuk-table__row
         %td.govuk-table__cell
-          = delivery_partner_organisation.name
+          = a11y_action_link delivery_partner_organisation.name, organisation_path(delivery_partner_organisation), t("table.cell.organisations.details"), ["govuk-link--no-visited-state"]
```

Comment:
What about "View [organisation name] organisation details"? Would need to extend the `a11y_action_link` functionality though, so might not be worth it

Reply:
This is a good suggestion, but I'm hesitant to make a change to this method here as it's used in a lot of places. I'm hoping this is usable enough. We'll be having an accessibility audit later in the year so we can revisit it then.